ws/src/main/webapp/WEB-INF/jsp/submit/verifyImport.jsp

```diff
@@ -92,7 +92,7 @@
 		<td class="table_header_row">Summary Statistics:</td>
 	</tr>
 	<tr valign="top">
-    	<td class="table_cell_left">Subjects Affected: <c:out value="${summaryStats.studySubjectCount}" /></td>
+    	<td class="table_cell_left">Participants Affected: <c:out value="${summaryStats.studySubjectCount}" /></td>

```

Comment:
For this file (verifyImport.jsp) this line is the only real diff. I don't know why there are so many other lines marked changed, even though there's no changes there.